### DISCUSSION TITLE
deployment in subdir

### DISCUSSION THREAD

**Discussion by Marx1st** [leihs/leihs#1264] [Q&A / Community Support] [answered]
Hi!

Is it possible to deploy into an subdirectory? Eg. https://mydomain.com/leihs ?

**Comment by DrTom** [comment, marked as answer]
No, not without changing the code and adding considerable complexity to it. 

I could imaging that one could try to do this entirely in the reverse proxy. But there is a lot of dynamic content and I believe this will be a problem. So I would strongly advise against trying this.  

(FYI @eins78 @nimaai)
